### PR TITLE
Flash click-to-run demo (fixes #366)

### LINKED ISSUE #366: Add simple demo for Flash Click-To-Run
We need to host a simple Flash CTR demo here that we can point to from Edge dev docs. Our usual CodePen host doesn't support Flash.

### PR DESCRIPTION
**Pull request by erikadoyle**
## What this PR does
Adding a simple demo for enabling Flash Click-To-Run. This isn't intended to be a showcase TestDrive with splash page etc. Just a simple demo we can't point to from the docs (since CodePen doesn't support Flash).

## Requirements

* [x] My PR follows all applicable accessibility requirements (See [`.github/ACCESSIBILITY_REQS.md`](https://github.com/MicrosoftEdge/Demos/blob/master/.github/ACCESSIBILITY_REQS.md)).

### CODE REVIEW

File: flashclicktorun/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    document.querySelector(".flashbox embed").style.display = "block";
+                }
+            });
+		</script>
+	</head>
+	<body>
+        <h1>Enabling Flash from Microsoft Edge</h1>
+        <div class="flashbox">
+            <div>Flash: <span class="status">Not Enabled</span></div>			
+            <div class="getflash" onclick="window.open('http://get.adobe.com/flashplayer', '_blank')">
```
Comment (molant) [inline flashclicktorun/index.html:41]: https please

File: flashclicktorun/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	<meta charset="utf-8">
+	<title>Flash Click-To-Play Demo</title>
+	<meta name="og:title" content="Flash Click-To-Play">
+	<meta name="description" content="Provide a seamless user experience on sites requiring Adobe Flash.">
+	<meta name="keywords" content="Misc, Flash">
+	<meta name="author" content="erikadoyle">
+    <link rel="stylesheet" href="https://edgeportal.blob.core.windows.net/media/demotemplate.css"/>
+	<link rel="stylesheet" href="styles/demo.css">
+		<script>
+            document.addEventListener("DOMContentLoaded", function() {
```
Comment (molant) [inline flashclicktorun/index.html:13]: Can we move this to a separate script?

File: flashclicktorun/index.html (hunk 1 above)
Comment (alrra) [reply flashclicktorun/index.html:41]: Let's move this inside the script as well, and [prevent the opening page from the possibility of abusing `window.opener`](https://mathiasbynens.github.io/rel-noopener/).

File: flashclicktorun/index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en-US">
+	<head>
+	<meta charset="utf-8">
+	<title>Flash Click-To-Play Demo</title>
+	<meta name="og:title" content="Flash Click-To-Play">
+	<meta name="description" content="Provide a seamless user experience on sites requiring Adobe Flash.">
+	<meta name="keywords" content="Misc, Flash">
+	<meta name="author" content="erikadoyle">
+    <link rel="stylesheet" href="https://edgeportal.blob.core.windows.net/media/demotemplate.css"/>
```
Comment (alrra) [inline flashclicktorun/index.html:10]: Remove Trailing `/`.

File: flashclicktorun/index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            });
+		</script>
+	</head>
+	<body>
+        <h1>Enabling Flash from Microsoft Edge</h1>
+        <div class="flashbox">
+            <div>Flash: <span class="status">Not Enabled</span></div>			
+            <div class="getflash" onclick="window.open('http://get.adobe.com/flashplayer', '_blank')">
+                <label for="flashlink">Click the image to display the browser prompt to enable Flash.</label>
+                <img id="flashlink" src="images/GetAdobeFlashPlayer_icon.png"/>
```
Comment (alrra) [inline flashclicktorun/index.html:43]: This image can be smaller:

**Comment by alrra** [comment]
@molant @erikadoyle Fixed a bunch of things and merged this with some modifications (
see: https://github.com/MicrosoftEdge/Demos/commit/bb64d184ff816ade1f0b2742ffb706a91ff2c59d & https://github.com/MicrosoftEdge/Demos/commit/d93ff9cde3302896e97bad128134d6c75426eef5). If there are other things that need to be address they can be done with a new PR.

**Comment by alrra** [comment]
> Fixed a bunch of things and merged this with some modifications.

See: https://github.com/MicrosoftEdge/Demos/commit/bb64d184ff816ade1f0b2742ffb706a91ff2c59d & https://github.com/MicrosoftEdge/Demos/commit/acd2ab4ddb33a46dda5582587f55e9cf86d5f1e9.

**Comment by alrra** [comment]
> https://github.com/MicrosoftEdge/Demos/commit/acd2ab4ddb33a46dda5582587f55e9cf86d5f1e9

Make that https://github.com/MicrosoftEdge/Demos/commit/d93ff9cde3302896e97bad128134d6c75426eef5 😶

**Comment by molant** [comment]
@erikadoyle do you want this demo live in the site? We will need a tile for that. Maybe @melanierichards can help with it.

**Comment by melanierichards** [comment]
@molant @erikadoyle can do! Re: tile

Thanks @melanierichards ! Let me check with Nathan Starr if this is something we want featured on the site (or not).

**Comment by erikadoyle** [comment]
Thanks so much @alrra and @molant!

**Pull request closed without merging**